streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java

```diff
@@ -598,9 +598,7 @@ public void updateHotParams(Application appParam) {
     ExecutionMode executionModeEnum = appParam.getExecutionModeEnum();
     Map<String, String> hotParams = new HashMap<>(0);
     if (ExecutionMode.YARN_APPLICATION.equals(executionModeEnum)) {
```

Comment:
`ExecutionMode.YARN_APPLICATION.equals(executionModeEnum)` need to be changed to:
`ExecutionMode.isYarnMode(executionModeEnum)` This is a historical bug

Reply:
oh, yarn-session execmode does not need to be set yarn queue, yarn-perjob is deprecated, but there should be many users who use, so. yarn-application and yarn-perjob need to be set yarn-queue

Reply:
> yarn-application and yarn-perjob need to be set yarn-queue

that's okay.